README.md

```diff
@@ -63,8 +63,11 @@ GITHUB_ID=47ec837fcbac1c6f1690
 GITHUB_SECRET=d59af0da231fc697d7dd8d3002ed1d9765e11c7f
 
 DUST_API=http://localhost:3001
+CONNECTORS_API=http://localhost:3002
 
 GA_TRACKING_ID=Foo
+
+DUST_CONNECTORS_SECRET=L3AhrnR3YuYEx6otwUnbzit29C1hTYnp
```

Comment:
remove this one for now as this is for developers running dust locally. They likely won't run `connectors` for now.